frontend/src/components/maps/leaflet/InventoryLayer.tsx

```diff
@@ -213,12 +213,11 @@ export const InventoryLayer: React.FC<InventoryLayerProps> = ({
         point => `${point?.properties.id}-${point?.properties.propertyTypeId}`,
       );
 
-      const hasBuildings = (property: any) => {
-        return items.find(
-          ({ properties }: any) =>
-            properties.propertyTypeId === 1 && properties.parcelId === property.parcelId,
-        );
-      };
+      /**
+       * Whether the land has buildings on it.
+       * @param property PIMS property
+       */
+      const hasBuildings = (property: IParcel | IBuilding) => false;
```

Comment:
Leaving function in as we'll probably need to go back to it.  For now it force all land to show up as pins.